api-reference/10 UI Components/dxDropDownEditor/1 Configuration/fieldAddons/afterTemplate.md

```diff
@@ -0,0 +1,18 @@
+---
+id: dxDropDownEditor.Options.fieldAddons.afterTemplate
+type: template
+---
+---
+##### shortDescription
+Specifies custom markup on the right of editor input fields.
```

Comment:
I want a transition here from the markup world (code) to the physical world (layout). In other words, markup doesn't appear on the right of editor. It defines content that appears on the right side of the editor.  

```suggestion
A custom markup that displays content to the right of the input field.
```